crates/bitwarden-core/src/key_management/crypto_client.rs

```diff
@@ -172,6 +172,13 @@ impl CryptoClient {
         derive_pin_user_key(&self.client, encrypted_pin)
     }
 
+    /// Generates a PRF-protected user key from the provided PRF secret. The result can be stored
+    /// and later used to initialize another client instance by using the PRF and the PRF key
+    /// with `initialize_user_crypto`.
+    pub fn derive_prf_key(&self, prf: B64) -> Result<RotateableKeySet, CryptoClientError> {
```

Comment:
```suggestion
    pub fn derive_prf_rotateable_keyset_for_user(&self, prf: B64) -> Result<RotateableKeySet, CryptoClientError> {
```

My suggestion here is very verbose, but "PRF key" alone suggests the "external key" (prf key) not the entire keyset.